### PR TITLE
Bug 483624 - Decorations for the www folder

### PR DESCRIPTION
**Pull request by misevskia**
Add globe icon decorator to top left quadrant of ${PROJECT_HOME}/www
folder in Hybrid Mobile Projects.

Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=483624
Signed-off-by: Angel Misevski <amisevsk@redhat.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=483624

File: plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/decorators/WWWFolderDecorator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		try {
+			if (folder.getProject().hasNature("org.eclipse.thym.core.HybridAppNature")
+				&& folder.getProjectRelativePath().toPortableString().equals("www")) {
+
+				decoration.addOverlay(wwwOverlay);
+			}
+		} catch (CoreException e) {
+			// CoreException is thrown if the project either does not exist,
+			// or is not open; neither should happen in this context.
+			e.printStackTrace();
```
Comment (gorkem) [inline plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/decorators/WWWFolderDecorator.java]: Can you please log this exception.

Comment (misevskia) [inline plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/decorators/WWWFolderDecorator.java]: Fixed now.

File: plugins/org.eclipse.thym.ui/plugin.xml [hunk 2]
```diff
@@ -304,4 +304,22 @@
...
+   <extension
+         point="org.eclipse.ui.decorators">
+      <decorator
+            id="org.eclipse.thym.ui.wwwDecorator"
+            label="%WWWDecorator.name"
+            class="org.eclipse.thym.ui.decorators.WWWFolderDecorator"
+            adaptable="true"
+            lightweight="true"
+            location="TOP_LEFT"
+            state="false">
```
Comment (gorkem) [inline plugins/org.eclipse.thym.ui/plugin.xml]: I think this decorator is not very invasive so it  is OK to enable this decorator by default.

Comment (misevskia) [inline plugins/org.eclipse.thym.ui/plugin.xml]: I agree, it only appears on one folder in each project, and shouldn't interfere with any other common decorators. I've changed it now.

**Pull request merged**